streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -286,7 +286,7 @@ abstract class DStream[T: ClassTag] (
     dependencies.foreach(_.validateAtStart())
 
     logInfo("Slide time = " + slideDuration)
```

Comment:
Thanks! I was thinking about it, but was worried to propose such changes as you might not like it :)